packages/vda-did-resolver/package.json

```diff
@@ -0,0 +1,102 @@
+{
+  "name": "vda-did-resolver",
```

Comment:
Should be @verida/vda-did-resolver.

Need to update references in other linked packages.